consul_config.py.ctmpl

```diff
@@ -0,0 +1,68 @@
+{{- define "KEY" -}}
+    {{ key (printf "docker-server-configs/LB/config.%s.json/%s" (env "DEPLOY_ENV") .) }}
+{{- end -}}
+
+
+DEBUG = False
+
+SECRET_KEY = '''{{template "KEY" "secret_key"}}'''
+
+{{if service "listenbrainz-db"}}
+{{with index (service "listenbrainz-db") 0}}
+SQLALCHEMY_DATABASE_URI = "postgresql://listenbrainz:listenbrainz@{{.Address}}:{{.Port}}/listenbrainz"
+MESSYBRAINZ_SQLALCHEMY_DATABASE_URI = "postgresql://messybrainz:messybrainz@{{.Address}}:{{.Port}}/messybrainz"
+POSTGRES_ADMIN_URI="postgresql://postgres@{{.Address}}:{{.Port}}/template1"
+{{end}}
+{{end}}
+
+# Not currently used
+MAX_POSTGRES_LISTEN_HISTORY = "-1"
+PG_QUERY_TIMEOUT = "3000"
+PG_ASYNC_LISTEN_COMMIT = False
+
+{{if service "listenbrainz-redis"}}
+{{with index (service "listenbrainz-redis") 0}}
```

Comment:
No -- service allows you to check if that service is up and running and if so, you can execute the "with" statement to get the IP/port of the service you want to use.